assets/cohere.yaml

```diff
@@ -163,7 +233,79 @@
   size:
     value: unknown
     explanation: >
-      The exact sizes of the generation models are unknown, but we know
+      The exact sizes of the embedding models are unknown, but we know
+      that they come in three sizes: small, medium, and large
+      [[Model Card]](https://docs.cohere.ai/representation-card).
+  analysis: >
+    The model's performance was analyzed on several safety benchmarks
+    [[Model Card]](https://docs.cohere.ai/representation-card).
+  # Construction
+  dependencies: [coheretext]
+  training_emissions:
+    value: unknown
+    explanation: The emissions of the models are unknown.
+  training_time:
+    value: unknown
+    explanation: The training time for the models are unknown.
+  training_hardware:
+    value: unknown
+    explanation: >
+      The training hardware wasn't explicitly announced, but it was
+      reported that Google Cloud teamed up with Cohere on a TPU partnership
+      [[TechCrunch Article]]
+      (https://techcrunch.com/2021/11/17/google-cloud-teams-up-with-nlp-startup-cohere-on-multi-year-partnership/).
+  quality_control: unknown
+  # Downstream
+  access:
+    value: limited
+    explanation: >
+      The model is available to the public through the Cohere Platform
+      [[Cohere Platform]](https://cohere.ai/).
+  license:
+    value: unknown
+    explanation: The model likely has a license specifically for Cohere's use.
+  intended_uses: >
+    The intended uses are stated as "estimating semantic similarity between two
+    sentences, choosing a sentence which is most likely to follow another
+    sentence, sentiment analysis, topic extraction, or categorizing user
+    feedback" on the Cohere model card
+    [[Model Card]](https://docs.cohere.ai/representation-card).
+  prohibited_uses: >
+    The usage of the model is bound by the Cohere usage guidelines
+    [[Usage Guidelines]](https://docs.cohere.ai/usage-guidelines).
+    A non-comprehensive list of specific application violating these guidelines
+    are: extraction of identity and demographic information, building
+    purposefully opaque text classification systems, and "building downstream
+    classifiers that serve as automated decision-making systems that have
+    real-world consequences on people, where those decisions are made without
+    a
+    human-in-the-loop"
+    [[Model Card]](https://docs.cohere.ai/representation-card).
+  monitoring: >
+    The usage of the model is monitored by Cohere
+    [[Model Card]](https://docs.cohere.ai/representation-card).
+  feedback:
+    value: unknown
+    explanation: >
+      There are no specific feedback mechanisms for the Generation model,
+      but a generic contact email is provided on Cohere website, which is
+      support at cohere.ai.
+
+- type: model
+  name: Embedding (Multilingual)
```

Comment:
should the dependencies be the same for both the English and Multilingual embeddings (i.e. both coheretext)?

Reply:
Likely more accurate to leave blank for now.